HomeWork18/HomeWork18/Dtos/EmployeeDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace HomeWork18.Dtos
+{
+    internal class EmployeeDto
+    {
+        [JsonProperty("name")]
+        public string Name { get; set; } = null!;
+
+        [JsonProperty("job")]
+        public string Job { get; set; } = null!;
+
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [JsonProperty("createdAt")]
+        public string? CreatedAt { get; set; }
```

Comment:
> What aboud DateTime instead of string?

in  this discription https://reqres.in/api-docs/#/default/put_users__id_ createAt string. So i think CreatedAt string too